proposals/lip-0050.md

```diff
@@ -46,7 +46,7 @@ We define the following constants:
 |---------------------------------|---------| -------|------------------------------------------------------------------------------------------------------------|
 | `MODULE_ID_LEGACY `             | uint32  | TBD    | Module ID of the Legacy module.                                                                            |
```

Comment:
We could already change ID type to `bytes` here (module ID and two command IDs).